Formula/tiv.rb

```diff
@@ -0,0 +1,23 @@
+class Tiv < Formula
+  desc "Display images in a terminal using block graphic characters"
+  homepage "https://github.com/stefanhaustein/TerminalImageViewer"
+  url "https://github.com/stefanhaustein/TerminalImageViewer/archive/refs/tags/v1.1.1.tar.gz"
+  sha256 "9a5f5c8688ef8db0e88dfcea6a1ae30da32268a7ab7972ff0de71955a75af0db"
+  license "Apache-2.0"
+  head "https://github.com/stefanhaustein/TerminalImageViewer.git", branch: "master"
+  depends_on "imagemagick"
+
+  uses_from_macos "curl" => :test
```

Comment:
Let's add some whitespace and force a newer GCC for linux
```suggestion
  head "https://github.com/stefanhaustein/TerminalImageViewer.git", branch: "master"

  depends_on "imagemagick"

  uses_from_macos "curl" => :test

  on_linux do
    depends_on "gcc"
  end

  fails_with gcc: "5"
```

Reply:
Done! Pretty sure it will fail unless it is at least 8 though